### PR TITLE
fix: Issue #628 Failed Cloud Build plan in 1-org using cloud build SA from TFV

### PR DESCRIPTION
**Pull request by ipv1337**
This fixed my issue #628.

### CODE REVIEW

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR, could you also add a google_folder_iam_member binding if `var.parent_folder != ""`?

**Comment by ipv1337** [comment]
> Thanks for the PR, could you also add a google_folder_iam_member binding if `var.parent_folder != ""`?

Let me know if the recent commit is what you are looking for @bharathkkb.

File: 0-bootstrap/main.tf [hunk 1]
```diff
@@ -176,6 +176,20 @@ data "google_project" "cloudbuild" {
   depends_on = [module.cloudbuild_bootstrap.csr_repos]
 }
 
+resource "google_organization_iam_member" "org_cb_sa_iam_viewer" {
+  count  = var.parent_folder == "" ? 1 : 0
+  org_id = var.org_id
+  role   = "roles/iam.organizationRoleViewer"
```
Comment (bharathkkb) [inline 0-bootstrap/main.tf:182]:
```suggestion
  role   = "roles/iam.securityReviewer"
```

File: 0-bootstrap/main.tf [hunk 2]
```diff
@@ -176,6 +176,20 @@ data "google_project" "cloudbuild" {
...
+  member = "serviceAccount:${data.google_project.cloudbuild.number}@cloudbuild.gserviceaccount.com"
+}
+
+resource "google_folder_iam_member" "org_cb_sa_iam_viewer" {
+  count  = var.parent_folder != "" ? 1 : 0
+  folder = var.parent_folder
+  role   = "roles/iam.organizationRoleViewer"
```
Comment (bharathkkb) [inline 0-bootstrap/main.tf:189]:
```suggestion
  role   = "roles/iam.securityReviewer"
```

**Review by bharathkkb — COMMENTED** [review]
It looks like we will actually need `roles/iam.securityReviewer` as [TFV will need to get IAM policy associated with any resource](https://github.com/GoogleCloudPlatform/terraform-validator/blob/8f1cc1a0d8d7feba66d8eb67f16467a3d930ab82/docs/user_guide.md#required-permissions) x-ref: https://github.com/terraform-google-modules/terraform-example-foundation/issues/546. It also looks like `organizationRoleViewer` can only be granted at the org level

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**